.travis.yml

```diff
@@ -22,7 +22,7 @@ deploy:
   api_key: $RUBYGEMS_API_KEY
   gem: trusona
   gemspec: trusona.gemspec
-  cleanup: true
```

Comment:
Oh ya that doesn't work even though stupid Travis complains about `skip_cleanup` being deprecated.

Reply:
it does work .. which is why the release did not happen .. it deleted the key during cleanup

Reply:
<img width="526" alt="Screen Shot 2020-02-04 at 10 54 55 AM" src="https://user-images.githubusercontent.com/49415226/73772117-d2e42b80-473c-11ea-99ef-64f356d1a364.png">

Reply:
oh hah. I swear I tried to switch it and stuff started failing - that's when I started switching them over to use the stages instead of `deploy`.